src/Sarif/RuntimeConditions.cs

```diff
@@ -52,6 +52,7 @@ public enum RuntimeConditions : uint
         ExceptionPostingLogFile = 0x40000,
 
         // Non-fatal conditions
```

Comment:
>Non [](http://example.com/codeflow?start=11&length=3)

Why isn't this a fatal condition? How do we proceed if the configured rules are not compatible?